app/src/main/java/com/rockthevote/grommet/data/db/dao/SessionDao.kt

```diff
@@ -17,7 +17,17 @@ interface SessionDao {
     @Query("SELECT * FROM session LIMIT 1")
     fun getCurrentSession(): Session?
 
+    @Transaction
+    @Query("SELECT * FROM session")
+    fun getSessionWithPartnerInfo(): LiveData<SessionWithPartnerInfo?>
+
     @Transaction
     @Query("SELECT * FROM session")
     fun getSessionWithRegistrations(): List<SessionWithRegistrations>
+
+    @Insert
```

Comment:
We might want an `onConflict` strategy specified here, especially if we're only using one session.

Reply:
i'll try adding it back, it may have been the suspend declaration i was using (removed it though)